docs/tutorials/10_advanced_parameter_sharing.ipynb

```diff
@@ -0,0 +1,909 @@
+{
```

Comment:
might make sense to add a comment to the other tutorials on setting and making params trainable on the lines of "If you're applications requires more flexiblity when setting / sharing parameters see tut 09 / 10"

Reply:
yes totally agree!